ui/app/src/translations/locales/en.json

```diff
@@ -295,10 +295,10 @@
   "publishingDashboard.confirmAllHelper": "Set the state for all selected items to \"Cancelled\"?",
   "publishingDashboard.confirmHelper": "Set the state for all selected items to \"Cancelled\"",
   "publishingDashboard.confirmHelperText": "Set item state to \"Cancelled\"?",
-  "publishingDashboard.environment": "Publishing Target",
+  "publishingDashboard.environment": "Publishing",
   "publishingDashboard.fetchPackagesFiles": "Fetch Packages Files",
   "publishingDashboard.filesList": "files list",
-  "publishingDashboard.filteredBy": "Showing: {state, select, all {} other {Status: {state}.}} {environment, select, all {} other {{environment} target.}} {path, select, none {} other {Filtered by {path}}}",
```

Comment:
Why are we rolling back these changes? We want to remove the word environment from this context and make it target/publishing target.